core/controllers/topics_and_skills_dashboard.py

```diff
@@ -174,6 +177,16 @@ def post(self):
             state_domain.SubtitledHtml.from_dict(explanation_dict))
         skill_services.save_new_skill(self.user_id, skill)
 
+        for filename in filenames:
+            image = self.request.get(filename)
+            file_format = html_validation_service.validate_image_and_filename(
```

Comment:
Raise InvalidInputException for validation errors

Reply:
Done